tests/stubtest_third_party.py

```diff
@@ -128,6 +134,81 @@ def run_stubtest(dist: Path, *, verbose: bool = False, specified_platforms_only:
     return True
 
 
+def setup_uwsgi_stubtest_command(dist: Path, venv_dir: Path, stubtest_cmd: list[str]) -> bool:
+    """Perform some black magic in order to run stubtest inside uWSGI.
+
+    We have to write the exit code from stubtest to a surrogate file
+    because uwsgi --pyrun does not exit with the exitcode from the
+    python script. We have a second wrapper script that passed the
+    arguments along to the uWSGI script and retrieves the exit code
+    from the file, so it behaves like running stubtest normally would.
+    """
```

Comment:
It wasn't immediately obvious to me that the scripts being generated would automatically be cleaned up:

```suggestion

    Both wrapper scripts are created inside `venv_dir`,
    which itself is a subdirectory inside a temporary directory,
    so both generated scripts will be cleaned up after this function
    has been executed.
    """
```